.ci/windows-steps.yaml

```diff
@@ -10,7 +10,6 @@ steps:
 - powershell: |
     nuget install OpenBLAS -o $(Agent.ToolsDirectory)
     nuget install boost -o $(Agent.ToolsDirectory) -Version 1.60.0
-    nuget install boost_unit_test_framework-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_program_options-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
```

Comment:
Nice to remove that!  I think we could also remove boost::program_options and boost::serialization too, unless the models repository is using them.  At the very least, mlpack itself is only using cereal and CLI11 (packaged internally).